web/src/user_card_popover.js

```diff
@@ -648,7 +648,7 @@ function register_click_handlers() {
     });
 
     $("body").on("click", ".user-card-popover-actions .narrow_to_messages_sent", (e) => {
-        const user_id = elem_to_user_id($(e.target).parents("ul"));
+        const user_id = elem_to_user_id($(e.target).closest(".user-card-popover-actions"));
         const email = people.get_by_user_id(user_id).email;
         hide_all();
         if (overlays.is_active()) {
```

Comment:
I'd rather we just do new click handlers rather than the reuse here; then the `overlays.is_active` check isn't necessary here, for example, and it's easier to verify things if independent components just have their own click handlers.

Reply:
Fixed.